beacon_node/src/cli.rs

```diff
@@ -1238,16 +1238,6 @@ pub fn cli_app() -> Command {
                 .action(ArgAction::Set)
                 .display_order(0)
         )
-        .arg(
-            Arg::new("disable-lock-timeouts")
```

Comment:
You gotta leave this part for a soft-deprecation. We should accept the flag, but log a warning. We _should not_ refuse to start if the flag is supplied until a few releases are done